### PR TITLE
Refactored time domain windowing, removed more stuff from spectrumdsp

### PR DESCRIPTION
**Pull request by JustAnEE**

### CODE REVIEW

File: Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/PhaseSpectrum.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+{
+   
+   return; 
+}
+
+void PhaseSpectrum::CalculateSpectrum()
+{
+   //const float fPhaseThreshold = 0.0f;
+
+   //!TODO: No idea what this code is even doing at the moment.
```
Comment (DanSop) [inline Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/PhaseSpectrum.cpp:15]: Interesting we should look into this....

File: Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/SpectralFactory.cpp [hunk 2]
```diff
@@ -17,10 +18,17 @@ SpectrumBase* SpectralFactory::Instantiate(SpectrumTypeEnum eSpectrumType_, int
       case DB_SPECTRUM:
```
Comment (DanSop) [inline Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/SpectralFactory.cpp:18]: how many spaces does this need?

File: Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/WindowTypes/BlackmanWindow.cpp [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+   InstantiateWindow();
+   return; 
+}
+
+
+void BlackmanWindow::InstantiateWindow()
+{
+   for (int k = 0; k < iMyBufferSize; k++)
+   {
+      pafMyWindow[k] = 0.42f - 0.5f * cosf(2.0f * 3.14159f * (float)k / ((float)(iMyBufferSize - 1)))
```
Comment (DanSop) [inline Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/WindowTypes/BlackmanWindow.cpp:17]: This is interesting.. any way we could clean this up?

**Comment by DanSop** [comment]
Overall well done! Lets try to keep the spacing and tabbing consistent throughout the project! Well done mr moroney

**Review by DanSop — APPROVED** [review]
Approved

File: Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/PhaseSpectrum.cpp (hunk 1 above)
Comment (JustAnEE) [reply Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/PhaseSpectrum.cpp:15]: !!
Calculating phase hard, code was not doing anything that made sense. Moved and commented out for now.

**Pull request merged**